content/applications/productivity/calendar.rst

```diff
@@ -9,3 +10,246 @@ Calendar
+
+  - :guilabel:`Timezone`: Select the timezone for which this meeting time is specified.
+  - :guilabel:`Repeat`: Select the recurring period of this meeting. Depending on what type of
+    recurrence has been selected, a subsequent field appears, in which users can indicate when the
+    meeting should recur. For example, if :guilabel:`Monthly` is selected as the :guilabel:`Repeat`
+    option, a new field appears, in which the user decides on what :guilabel:`Day of Month` the
+    meeting should recur.
+  - :guilabel:`Until`: Select the limited :guilabel:`Number of repetitions` this meeting should
+    recur, the :guilabel:`End date` of when the recurrences should stop, or if the meetings should
+    recur :guilabel:`Forever`.
+- :guilabel:`Tags`: Add tags to the event, like `Customer Meeting` or `Internal Meeting`. These can
+  be searched and filtered in the **Calendar** app when organizing multiple events.
+- :guilabel:`Appointment`: Link existing or new appointments. These can be configured through the
+  *Share Availabilities* button from the main **Calendar** dashboard.
+- :guilabel:`Privacy`: Toggle between visibility options to control who can view the event.
+- :guilabel:`Organizer`: This is defaulted to the current Odoo user. Select a new one from
+  existing contacts, or create and edit a new contact.
+- :guilabel:`Description`: Add additional information or details about the meeting.
+- :guilabel:`Reminders`: Select notification options to send to attendees. Choose a default
+  notification, or configure new reminders.
+
+Coordinate with teams
+---------------------
+
+When scheduling an event for multiple users, on the **Calendar** app dashboard, tick the checkbox
+next to :guilabel:`Attendees` to view team members' availability. Tick (or untick) the checkbox next
+to listed users to show (or hide) individual calendars.
+
+.. image:: calendar/calendar-attendees.png
+   :align: center
+   :alt: View of Attendees section on Calendar app.
+
+Share Availabilities
+--------------------
+
+On the **Calendar** app main dashboard, click the :guilabel:`Share Availabilities`
+button at the top of the page. Select the available times and dates on the calendar to add them as
+options in the invitation. Once availability has been selected, click the :guilabel:`Open` button
+to navigate to the associated meeting.
+
+.. image:: calendar/calendar-meeting-share-availability.png
+   :align: center
+   :alt: Share availability window on Calendar app.
+
+.. note::
+   Within the :guilabel:`Share Availabilities` feature, selecting date and times is only possible
+   on the *Day*, *Week*, and *Year* calendar views.
+
+On the meeting form, users have several configuration options:
+
+In the :guilabel:`Scheduling` field, select a minimum hour window to ensure appointments are
+confirmed a specified amount of time in advance. For example, select :guilabel:`01:00` to require
+attendees to confirm at least one hour before their appointment time.
```

Comment:
Semantic nitpick– there is nothing to select in this field, its all typing inputs :)
```suggestion
In the :guilabel:`Scheduling` field, set a minimum hour window to ensure appointments are confirmed
a specified amount of time in advance. For example, set `01:00` to require attendees to confirm at
least one hour before their appointment time.
```